static/i18n/ru.lua

```diff
@@ -32,7 +32,7 @@ return {
     upload_button_header = "Выберите пакет для загрузки",
     upload_package = "Загрузить пакет",
     upload_success = "Пакет %s загружен успешно",
-    user_created = "Пользователь создан",
+    user_created = "Thanks for registering! Please check your email address to verify your account.",
```

Comment:
Intentionally set it to the english in the other translation files? I mean, not a huge issue since we're probably changing translation setups I suppose.

Reply:
Yeah, so we remember to update the translation too